### ISSUE TITLE
I don’t know why I can’t download the video

### ISSUE THREAD

**Issue by hezhushizaishi** [Tusko/vimeo-private-downloader#52]
Hello everyone, today I was preparing to download a video as usual, but I was prompted that the json was incorrect. I have confirmed that my operation was correct, and this is the URL I obtained:
https://179vod-adaptive.akamaized.net/exp=1714915362~acl=%2Fefcf1c6a-82c3-4629-82e1-e658a1cda6f2%2F%2A~hmac=60e86ad7793c5342198eee12d428f5f5e5a078d845ef4ee8cf5eda440c5e659a/efcf1c6a-82c3-4629-82e1-e658a1cda6f2/sep/video/18d065ce,46330b00,639288d3,a7f36529,d6628b67/audio/4682c1ac,96058176,a8449bea/master.json?base64_init=1&query_string_ranges=1
This is the error message: 
![1234567](https://github.com/Tusko/vimeo-private-downloader/assets/122086865/1763afd8-dd04-47fb-8bfe-af0913038fb6)
 I noticed that the URL I obtained today seems different from the ones I used to get. I was still able to use it on April 20th. I don't know if Vimeo has changed their format. If any of you friends have encountered the same issue and know how to resolve it, please let me know. I would be sincerely grateful.

**Comment by Tusko** [comment]
@hezhushizaishi 
1. remove everything after `?` `base64_init=1&query_string_ranges=1`
2. looks like the link is expired

**Comment by hezhushizaishi** [comment]
> @hezhushizaishi
> 
> 1. remove everything after `?` `base64_init=1&query_string_ranges=1`
> 2. looks like the link is expired

I want to express my sincere gratitude for your attention to my problem. I'm pleased to share that I've finally found a solution to the issue that has been troubling me for nearly two months.
Initially, I suspected the problem might be due to changes in Vimeo's format or other technical updates. However, after a series of trials, I discovered that the root cause was actually related to my network settings.
The breakthrough came when I enabled the TUN mode in my proxy software. Surprisingly, this simple change resolved the JSON error I had been encountering. Not only did it allow your software to function normally again, but it also fixed similar issues I was experiencing with other tools.
This experience has been quite enlightening. It helped me realize that the problem wasn't with your software or Vimeo's website, but rather with my local network configuration. The TUN mode seems to have addressed some network restrictions or compatibility issues that were causing the errors.
I truly appreciate your patience and support throughout this process. Your software continues to be an invaluable tool for me, and I'm relieved to have it working properly again.
Thank you once again for your excellent work and for maintaining such a useful
